lib/src/Base/Stat/openturns/LatentVariableModel.hxx

```diff
@@ -0,0 +1,137 @@
+//                                               -*- C++ -*-
+/**
+ *
+ *  Copyright 2005-2023 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_LATENTVARIABLEMODEL_HXX
+#define OPENTURNS_LATENTVARIABLEMODEL_HXX
+
+#include "openturns/CovarianceModelImplementation.hxx"
+#include "openturns/SquaredExponential.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class LatentVariableModel
+ */
+
+class OT_API LatentVariableModel
+  : public CovarianceModelImplementation
+{
+
+  CLASSNAME
+
+public:
+
+  /** Default constructor based on levels*/
+  explicit LatentVariableModel(const UnsignedInteger nLevels = 2);
+
+  /** Parameters constructor */
+  LatentVariableModel(const UnsignedInteger nLevels,
+						 const UnsignedInteger latentDim);
```

Comment:
both nLevels & latentDim could have default values so only one ctor is needed:
LatentVariableModel(const UnsignedInteger nLevels = 2, const UnsignedInteger latentDim = 2);